sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -220,4 +242,5 @@ class JsonFunctionsSuite extends QueryTest with SharedSQLContext {
     assert(errMsg2.getMessage.startsWith(
       "A type of keys and values in map() must be string, but got"))
   }
+
```

Comment:
I found a newline at the end of tests prevent a conflict in some cases (_if I am not mistaken_). I am happy to revert this change back if anyone is sure that it is uesless or feel like it is an unrelated change.